.github/workflows/build.yml

```diff
@@ -1,6 +1,7 @@
 name: Build
 
 on:
+  workflow_dispatch: {}
```

Comment:
Doesn't it need the branch info on which branch it is getting triggered?

Reply:
yes it gets that from `context.ref`